src/main/scala/com/netflix/edda/aws/AwsCrawlers.scala

```diff
@@ -160,9 +156,13 @@ abstract class AwsIterator extends Iterator[Seq[Record]] {
 class AwsAddressCrawler(val name: String, val ctx: AwsCrawler.Context) extends Crawler {
   val request = new DescribeAddressesRequest
 
-  override def doCrawl()(implicit req: RequestId) =
-    backoffRequest { ctx.awsClient.ec2.describeAddresses(request).getAddresses.asScala.map(
+  private def doNonAssumeCrawl()(implicit req: RequestId) =
+    backoffRequest { ctx.awsClient.ec2().describeAddresses(request).getAddresses.asScala.map(
       item => Record(item.getPublicIp, ctx.beanMapper(item))) }.toSeq
+
+  override def doCrawl()(implicit req: RequestId) =
+    backoffRequest { ctx.awsClient.ec2(true).describeAddresses(request).getAddresses.asScala.map(
+      item => Record(item.getPublicIp, ctx.beanMapper(item))) }.toSeq ++ (if(assumeRoleEnabled) doNonAssumeCrawl() else Seq.empty[Record])
```

Comment:
When `assumeRoleEnabled = true`, two kind of `Crawl()` is done. 
* `doCrawl()` pulls data from account where edda is hosted
* `doNonAssumeCrawl()` pulls data from account of ARN given

Code can be optimized bit more. I shall do it and push back